src/components/Tag.js

```diff
@@ -2,23 +2,24 @@ import React from 'react'
 import styled from 'styled-components'
 import Typography from './Typography'
 import get from 'utils/get'
+import hexToRgb from 'utils/hexToRgb'
 
 const TagWrapper = styled.div`
   color: ${get('colors.ashes')};
-  border: 1px solid ${get('colors.dodgerBlue')};
+  border: 1px solid ${p => get(p.$color)};
   border-radius: 8px;
   padding: 5px;
   display: inline-flex;
   align-items: center;
   justify-content: center;
-  background-color: ${get('colors.luckyPoint')};
+  background-color: ${p => hexToRgb(get(p.$color), 0.2)};
   height: 30px;
```

Comment:
I feel like there is a sizing problem here on smaller devices (it was something like a width of 800px)
<img width="731" alt="Screenshot 2022-11-03 at 10 16 39" src="https://user-images.githubusercontent.com/23705562/199687678-8fa394b3-1c6a-4218-838e-c3e4414ffbb9.png">

Reply:
Nice catch! ❤️
(done)